app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/onboarding/email/email_verified.scala.html

```diff
@@ -46,29 +45,28 @@
  else None
 }
 
-@mainTemplate(title = title, userType = request.userType, accountMenu = menu) {
+@layout(title, accountMenu = menu) {
 
-  @caption(messages(emailJourneyType.captionMessageKey))
+<span class="govuk-caption-xl">@messages(emailJourneyType.captionMessageKey)</span>
+<h1 class="govuk-heading-xl">@title</h1>
 
-  @pageHeading(title)
-
-  <p class="lede">
+  <p class="govuk-body govuk-!-margin-top-0 govuk-!-margin-bottom-3">
```

Comment:
https://design-system.service.gov.uk/styles/typography/#lead-paragraph this shows us the equivalent of the old `lede` class is `govuk-body-l`, this would probably change your AT update also